### PR TITLE
Delete not sorting signs in title (#1156)

### PR DESCRIPTION
**Pull request by TobiasNx**
See #1156

### CODE REVIEW

File: src/main/resources/alma/common/fields.xml [hunk 1]
```diff
@@ -675,7 +680,7 @@
           <constant value="Aufgegangen in" />
         </data>
         <data source="78505.t">
-          <constant value="Teilweise aufgegangen in" />
+        <any flushWith="700[013] ">
         </data>
```
Comment (dr0i) [inline src/main/resources/alma/common/fields.xml:684]: Not sure what this field has to do with this issue? Also formatting.

File: src/main/resources/alma/common/fields.xml [hunk 2]
```diff
@@ -365,11 +365,16 @@
     </data>
   </entity>
 	-->
-  <call-macro name="regex-del-punctuation-end" dsource="245??.a" dname="title" />
-  <data name="title" source="245??.p">
+  <call-macro name="regex-del-punctuation-end" dsource="245??.a" dname="@title" />
+  <data name="@title" source="245??.p">
     <occurrence only="1" />
     <replace pattern="\s?\.?\s?$" with="" />
   </data>
```
Comment (dr0i) [inline src/main/resources/alma/common/fields.xml:372]: Not sure why introduce a variable here. You could also simply add the new "replaces" here.

**Review by dr0i — CHANGES_REQUESTED** [review]

File: src/main/resources/alma/common/fields.xml (hunk 2 above)
Comment (TobiasNx) [reply src/main/resources/alma/common/fields.xml:372]: There are two sources of the title. I wanted to insert the use the replacement only once. 245.p is used if 245 .a does not exist.

File: src/main/resources/alma/common/fields.xml (hunk 1 above)
Comment (TobiasNx) [reply src/main/resources/alma/common/fields.xml:684]: you are right, this can be deleted.

File: src/main/resources/alma/common/fields.xml (hunk 2 above)
Comment (dr0i) [reply src/main/resources/alma/common/fields.xml:372]: Ah, yeah I see.So leave that as it is.

**Comment by TobiasNx** [comment]
Fix the inserted error.

**Comment by dr0i** [comment]
Merged. To be deployed.

**Pull request merged**